Sources/PackageCollectionsModel/Formats/v1.md

```diff
@@ -0,0 +1,235 @@
+* `versions`: An array of version objects representing the most recent and/or relevant releases of the package.
+
+#### Add versions to a package
+
+A version object has metadata extracted from `Package.swift` and optionally additional metadata from other sources:
+
+* `version`: The semantic version string.
+* `manifests`: A map of manifests by tools version. The keys are (semantic) tools version (more on this below), while the values are:
+    * `toolsVersion`: The Swift tools version specified in the manifest.
+    * `packageName`: The name of the package.
+    * `targets`: An array of the package version's targets.
+        * `name`: The target name.
+        * `moduleName`: The module name if this target can be imported as a module. **Optional.**
+    * `products`: An array of the package version's products.
+        * `name`: The product name.
+        * `type`: The product type. This must have the same JSON representation as SwiftPM's `PackageModel.ProductType`.
+        * `target`: An array of the product’s targets.
+    * `minimumPlatformVersions`: An array of the package version’s supported platforms specified in `Package.swift`. **Optional.** 
+
+```json
+{
+  "5.2": {
+    "toolsVersion": "5.2",
+    "packageName": "MyPackage",
+    "targets": [
+      {
+        "name": "MyTarget",
+        "moduleName": "MyTarget"
+      }
+    ],
+    "products": [
+      {
+        "name": "MyProduct",
+        "type": {
+          "library": ["automatic"]
+        },
+        "targets": ["MyTarget"]
+      }
+    ],
+    "minimumPlatformVersions": [
+      {
+        "name": "macOS",
+        "version": "10.15"
+      }
+    ]
+  }
+}
+```
+
+* `verifiedCompatibility`: An array of compatible platforms and Swift versions that has been tested and verified for. Valid platform names include `macOS`, `iOS`, `tvOS`, `watchOS`, `Linux`, `Android`, and `Windows`. Swift version should be semantic version string and as specific as possible. **Optional.**
+
+```json
+{
+  "platform": {
+    "name": "macOS"
+  },
+  "swiftVersion": "5.3.2"
+}
+```
+
+* `license`: The package version's license. **Optional.**
+    * `url`: The URL of the license file.
+    * `name`: License name. [SPDX identifier](https://spdx.org/licenses/) (e.g., `Apache-2.0`, `MIT`, etc.) preferred. Omit if unknown. **Optional.**
+
+##### Version-specific manifests
+
+Package collection generators should include data from the "default" manifest `Package.swift` as well as [version-specific manifest(s)](https://github.com/apple/swift-package-manager/blob/main/Documentation/Usage.md#version-specific-manifest-selection).
+
+The keys of the `manifests` map are tools (semantic) versions:
+* For `Package.swift`, the tools version specified in `Package.swift` should be used.
+* For version-specific manifests, the tools version specified in the filename should be used. For example, for `Package@swift-4.2.swift` it would be `4.2`.
```

Comment:
Should there be mention here that the tools version in the manifest is expected to be the same as the one in the filename?